addons/phone_validation/tests/test_phonenumbers_blacklist.py

```diff
@@ -13,3 +13,10 @@ def test_sanitize_search(self):
         res = blacklist.search([('number', 'in', ['+917 5896 32587'])])
 
         self.assertEqual(len(res), 1, "There should be one result")
+
+    def test_MX_blacklist(self):
+        """ Test that we can add a MX number to the blacklist """
+        blacklist = self.env['phone.blacklist']
+        blacklist.create({'number': '+527201020716'})
+        res = blacklist.search([('number', 'in', ['+527201020716'])])
+        self.assertEqual(len(res), 1, "There should be one result")
```

Comment:
This seem redundant with the test in addons/phone_validation/tests/test_phonenumbers_patch.py since it will try to parse it.

Reply:
Yup, thought the same, I added it because the ticket was specifically pointing this issue but I second you, will wait to see what the rest thinks. 🙂